lib/cli.js

```diff
@@ -432,6 +461,10 @@ module.exports = async (argv) => {
 		optionsBase.disableDefaultIgnores = cli.flags.disableDefaultIgnores;
 	}
 
+	if (cli.flags.ext) {
+		optionsBase.extensions = cli.flags.ext.replace(/\./g, '').split(',');
```

Comment:
[suggest] How about normalizing a comma-separated string like this?

- Trim whitespaces
- Filter empty strings

```suggestion
		optionsBase.extensions = cli.flags.ext.replace(/\./g, '').split(',').map(s => s.trim()).filter(Boolean);
```

Reply:
> Trim whitespaces

This is a flag argument so you shouldn't allow whitespace in between extensions.
i.e. `--ext .css,   .scss` shouldn't be allowed

> Filter empty strings

We maybe need a test for `--ext` followed by nothing.

Reply:
> This is a flag argument so you shouldn't allow whitespace in between extensions.

What about the case of `--ext=".css, .scss,"` using quotes?

Reply:
I think it shouldn't be allowed. Since _multiple_ is not supported, Id like to keep it simple.
But we maybe need a test for  dangling `,`.

- [ ] dangling `,`
- [ ] lone `--ext`

Reply:
I think it shouldn't be allowed. Since _multiple_ is not supported, Id like to keep it simple.
But we maybe need a test for a dangling `,`.

- [ ] dangling `,`
- [ ] lone `--ext`

Reply:
@ybiquitous Iv checked the case _lone --ext_ and AFAIK meow doesn't provide a way to know whether the flag is being passed or not when using a default—even using the `isRequired` callback.

I am gonna add a fix for _dangling `,`_ while I am at it Ill add your `map` even if I think it shouldn't be allowed; it doesn't hurt since we currently don't have error messages in place for the case `--ext=".css, .scss"`.